azure-pipelines/pull-request-validation/pr-msal.yml

```diff
@@ -7,13 +7,13 @@ name: $(date:yyyyMMdd)$(rev:.r)
 
 variables:
 - name: BuildParameters.jdkVersion
-  value: 1.8
+  value: 1.11
 - name: BuildParameters.jdkArchitecture
   value: x64
 - name: BuildParameters.javaHomeSelection
   value: JDKVersion
 - name: robolectricSdkVersion
-  value: 28
+  value: 33
```

Comment:
We have pipeline variable to set robolectric version (see- ProbolectricSdkVersion), please check if that needs to be updated as well.

Reply:
oops.. this the pipeline variable itself that you are modifying :)